pkg/chaosdaemon/util.go

```diff
@@ -120,3 +122,33 @@ func withNetNS(ctx context.Context, nsPath string, cmd string, args ...string) *
 
 	return exec.CommandContext(ctx, "nsenter", args...)
 }
+
+// ContainerKillByContainerID kills container according to container id
+func (c DockerClient) ContainerKillByContainerID(ctx context.Context, containerID string) error {
+	if containerID[0:len(dockerProtocolPrefix)] != dockerProtocolPrefix {
```

Comment:
If the `containerID` is empty, the program will panic here.  [line 72](https://github.com/pingcap/chaos-mesh/blob/master/pkg/chaosdaemon/util.go#L70) has the same issue.